ast/misc.go

```diff
@@ -2407,6 +2409,11 @@ const (
 	BRIEOptionCSVSeparator
 	BRIEOptionCSVTrimLastSeparators
 
+	// BRIEOptionLevel
+	BRIEOptionLevelOff BRIEOptionLevel = iota + 2
+	BRIEOptionLevelOptional
+	BRIEOptionLevelRequired
```

Comment:
why starts with `iota + 2` 🤔 

`checksum = 0` should mean OFF and `checksum = 1` should mean REQUIRED.

Reply:
`CHECKSUM` is shared by `RESTORE`. How about letting value of `CHECKSUM` resume to `0`/`1` in `RESTORE` to keep same with `ONLINE`, and let it resume to `OFF`/`REQUIRED`/`OPTIONAL` in `IMPORT`

Reply:
perhaps in BACKUP and RESTORE using `CHECKSUM = OPTIONAL` would throw an error.